### PR TITLE
CDPT-471 Remove useless secret from job deployment

### PR DESCRIPTION
**Pull request by digitalali-moj**
## Description
remove useless secret from deployment file

## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
* [x] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`

### CODE REVIEW

**Comment by vertism** [comment]
Looks good if these values aren't needed, but can you clarify why they are useless as I can see some of them mentioned in the README.

File: k8s-deploy/development/migration_job.yaml [hunk 1]
```diff
@@ -27,11 +22,6 @@ spec:
                 secretKeyRef:
                   key: DEVISE_SECRET
                   name: pq-secrets
-            - name: DEVISE_SENDER
-              valueFrom:
-                secretKeyRef:
-                  key: DEVISE_SENDER
-                  name: pq-secrets
```
Comment (Polsonby) [inline k8s-deploy/development/migration_job.yaml:34]: I think this is a valid ENV VAR, we might want to set it to something sensible if it's not - sets the "from" email address for auth emails sent by Devise, e.g. password reset emails?  
image

**Review by Polsonby — COMMENTED** [review]
All looks good, thanks for this, but one thing caught my eye, maybe we might want to keep that one

File: k8s-deploy/development/migration_job.yaml (hunk 1 above)
Comment (digitalali-moj) [reply k8s-deploy/development/migration_job.yaml:34]: It all goes through gov uk notify and the email address sent from is pq.tracker@notifications.service.gov.uk. The reply-to probably needs to be configured on Notify as it's a GSI address at the moment. I'll add a ticket for that. The Devise sender wasn't used before it was set to 'x'.

**Pull request merged**